packages/integration-tests/tests/api/tokens.test.ts

```diff
@@ -112,6 +112,51 @@ describe("Tokens", () => {
           expect(response.body.items[0]).toStrictEqual(expect.objectContaining({ type: TransactionsType.transfer }));
         });
       });
+
+      //id1803
+      it("Verify the response via /tokens/{address}/transfers", async () => {
+        await helper.retryTestAction(async () => {
+          l2Token = await helper.getStringFromFile(bufferFile + Buffer.customToken);
+          const emptyWallet = await helper.getStringFromFile(bufferFile + Buffer.emptyWalletAddress);
+          const paymaster = await helper.getStringFromFile(bufferFile + Buffer.paymaster);
+          txHash = await helper.getStringFromFile(bufferFile + Buffer.paymasterTx);
+          const apiRoute = `/tokens/${l2Token}/transfers?page=1&limit=10`;
+          const decapitalizedTokenAddress = l2Token.toLowerCase();
+          response = await helper.performGETrequest(apiRoute);
+
+          expect(response.status).toBe(200);
+          expect(response.body.items[0]).toStrictEqual(expect.objectContaining({ from: emptyWallet }));
```

Comment:
did you try, is it possible to send something from an empty wallet? is it really empty?